scripts/build.rs

```diff
@@ -50,11 +50,17 @@ fn cmake_build() {
     }
 
     // By default enable schannel on windows, unless openssl feature is selected.
-    if cfg!(windows) && !cfg!(feature = "quictls") {
+    if cfg!(feature = "quictls") {
+        config.define("QUIC_TLS_LIB", "quictls");
+    } else if cfg!(feature = "openssl") {
+        config.define("QUIC_TLS_LIB", "openssl");
+    } else if cfg!(windows) {
         config.define("QUIC_TLS_LIB", "schannel");
     } else {
-        config.define("QUIC_TLS_LIB", "quictls");
+        // Default to openssl
+        config.define("QUIC_TLS_LIB", "openssl");
```

Comment:
I'm not sure we're ready to default to this until the correct version is widely available in the distros.